src/components/HackerInfo/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { Button, Typography } from '@ht6/react-ui';
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+import { Navigate } from 'react-router-dom';
+
+import DevpostLogo from '../../assets/devpost.png';
+import DiscordLogo from '../../assets/discord.png';
+import HopinLogo from '../../assets/hopin.png';
+import useAuth from '../../components/Authentication/context';
+import { ServerResponse, useRequest } from '../../utils/useRequest';
+import IconLink from '../IconLink';
+
+import styles from './HackerInfo.module.scss';
+
+function HackerInfo() {
+  const [isRsvpEnabled, setIsRsvpEnabled] = useState(true);
+  const { makeRequest } = useRequest<ServerResponse>('/api/action/rsvp');
+  const authCtx = useAuth();
+
+  if (!authCtx.isAuthenticated) {
+    return null;
+  }
+
+  const email = authCtx.user.email;
+  const userConfirmed = authCtx.user.status.confirmed;
+
+  return (
+    <div className={styles.container}>
+      {!userConfirmed && <Navigate replace to='/invite' />}
```

Comment:
Redirects if user has already confirmed their attendance.

Is this the correct user status variable to use? I saw a lot of similar ones like `isRSVPOpen` or `accepted/declined`?

Reply:
Hmm it should be fine, i can look into this some more :p